index.html

```diff
@@ -46,717 +46,789 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
-					Canvas shortly following the lecture.
-				</p>
-				<p><b>Honor Code</b>: Students are free to form study groups and may discuss homework in groups.
-					However, each student must write down the solutions and code from scratch independently, and without
-					referring to any written notes from the joint session. When debugging code together, you are only
-					allowed to look at the input-output behavior of each other's programs and not the code itself.
-					In other words, each student must understand the solution well enough in order to reconstruct it by
-					him/herself. It is an honor code violation to copy, refer to, or look at written or code solutions
-					from a previous year, including but not limited to: official solutions from a previous year,
-					solutions posted online, and solutions you or someone else may have written up in a previous year.
-					Furthermore, it is an honor code violation to post your assignment solutions online, such as on a
-					public git repo. For more details about honor code, see <a
-						href="https://communitystandards.stanford.edu/policies-and-guidance/honor-code">The Stanford
-						Honor
-						Code</a> and <a
-						href="https://web.stanford.edu/class/archive/cs/cs106b/cs106b.1164/handouts/honor-code.pdf">The
-						Stanford Honor Code Pertaining to CS Courses</a>.
-				</p>
-			</div>
-
-            <!--
+        <div id="course-policies">
+            <h3> Grading and Course Policies </h3>
+            <p><b>Homeworks</b> (50&#37;): There are four graded homework assignments. To provide some
+                flexibility, the lowest scoring homework for each student will be worth 5&#37; of the grade,
+                while the remaining three will be worth 15&#37; of the grade. Assignments will require
+                training neural networks in PyTorch. All assignments are due on Gradescope at 11:59 pm
+                Pacific Time on the respective due date.
+            </p>
+            <p><b>Project</b> (50&#37;): There's a research-level project of your choice. You may form groups of 1-3
+                students to complete the project, and you are encouraged to start early! Project guidelines will
+                be posted in the first week of the class.
+            </p>
+            <p><b>Late Days</b>: You have 6 total late days across homeworks and project deliverables (anything worth
+                a grade), except for the project poster. You may use a maximum of 2 late days for any single assignment.
+                Late days used for group projects apply to all members of the group.
+            </p>
+            <p><b>Lecture Attendance</b>: While we do not require lecture attendance, students are encouraged to
+                join the live lecture. To accommodate various circumstances, we will be live-streaming the in-person
+                lecture via a zoom link on canvas. <!-- Students joining the zoom live stream will be able to ask
```

Comment:
We won't have a zoom link -- it's on panopto. It'll be on canvas though.  Could you revise?